### ISSUE TITLE
"Next" link from user_guide/io.rst goes to read_sql_table API page

### ISSUE THREAD

**Issue by TomAugspurger** [pandas-dev/pandas#30332]
I think that's because we include a `:toctree:` on line 4,832

```
.. autosummary::
    :toctree: ../reference/api/

    read_sql_table
    read_sql_query
    read_sql
    DataFrame.to_sql
```

We likely don't need that `toctree`. Deleting it may fix the issue.

To check, we probably need to do a full doc build.

**Comment by TomAugspurger** [comment]
Ideally the "Next" button would go to the next page in the User Guide toctree, "Indexing and selecting data".

**Comment by souvik3333** [comment]
Hi, Can I do it?

**Comment by tniharika** [comment]
Hi,

I would like to work on this

**Comment by TomAugspurger** [comment]
That'd be great if either of you could take this. Let's give preference to @souvik3333 since they commented first, though please comment if you can't work on it anymore. I believe you can self-assign an issue by posting "take".

**Comment by souvik3333** [comment]
take

**Comment by souvik3333** [comment]
@TomAugspurger I have updated this issue on document Version 0.26.0.dev0+1351.g70a083f04.dirty but in pydata.org version 0.25.3 is available. Should I do anything about it?

**Comment by TomAugspurger** [comment]
No, we won't update the old docs. When your PR is merged, it'll automatically be deployed to dev.pandas.io shortly after.

**Comment by WillAyd** [comment]
closed via #30348
